scripts/globals/effects/hagakure.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+--
+--     EFFECT_HAGAKURE
+--
+-----------------------------------
+
+require("scripts/globals/status");
+
+
+function onEffectGain(target,effect)
+    target:addMod(MOD_SAVETP,400);
+    target:addMod(MOD_TP_BONUS,1000);
+end;
+
+
+function onEffectTick(target,effect)
+end;
+
```

Comment:
you've got extara newlines, just fyi (wouldn't stop me from merging but while you are fixing that indent, may as well fix this)

Reply:
code/sql looks good